Resources/Prototypes/_LostParadise/Entities/Structures/Wallmounts/posters.yml

```
@@ -45,4 +45,74 @@
+  description: "It's like the first one, but with more funservice."
+  components:
+  - type: Sprite
+    state: lustfulvulp-basin
+
+- type: entity
+  parent: LPPPosterBase
+  id: LPPPosterTajaranHalloween
+  name: "Halloween tajaran"
+  description: "Trick or treet as they say..."
+  components:
+  - type: Sprite
```

yamllint

[error] wrong indentation: expected at least 3

(indentation)